debug/debug.go

```diff
@@ -50,7 +67,7 @@ func getUname() string {
 func Initialize() {
 	err := os.MkdirAll(LogDirectory, 0750)
 	if err != nil {
-		RecoverPrettyPanic = false
+		RecoverPrettyPanic = true
```

Comment:
This `if` happens when the log directory doesn't exist, so everything needs to be disabled. Enabling by default should probably be in the variable declaration